ansible/roles/prosody/files/mod_muc_webhooks.lua

```diff
@@ -496,6 +499,26 @@ function handle_occupant_access(event, event_type)
     end
 end
 
+function decorate_payload_with_flip(payload, occupant_nick, main_room, final_event_type)
+    if final_event_type == PARTICIPANT_JOINED then
+        local flip_participant_nick = main_room._data and main_room._data.flip_participant_nick
+        if occupant_nick and flip_participant_nick and flip_participant_nick == occupant_nick then
+            module:log("info", "Decorate participant joined event with flip for occupant nick %s", flip_participant_nick)
```

Comment:
do we want to log this information on info?

Reply:
i checked and I see the same in mod_muc_events

Reply:
We were blaming at some point flip for some problems and I think these logs are useful.